data/core/units/elves/Archer.cfg

```diff
@@ -16,7 +16,7 @@
     advances_to=Elvish Ranger,Elvish Marksman
     cost=17
     usage=archer
-    description= _ "Elves have always had a reputation for archery, a skill that, for many reasons, comes naturally to them. Even the frail of body can be deadly with a bow in hand, and in times of war, many will take up this weapon. Though their relative inexperience with combat leaves them somewhat vulnerable, their natural grace ensures that they can best any of humanity’s fresh recruits."
+    description= _ "As primarily foragers and hunters, most elves learn to become proficient archers from a young age. Besides being only a practical skill, archery is also a common pasttime and many competitions are held in sport for the entertainment of spectators and participants alike. This ability is readily turned to battle in times of war, where many elves will wield bows as their weapons of choice. Though not as sturdy as their human or orc counterparts, Elvish archers are still effective combatants especially when fighting from the safety of their forests."
```

Comment:
You might want to consider adding a comma before "especially" in the last sentence.